ncm-ceph/src/main/perl/Ceph/Jewel.pm

```diff
@@ -0,0 +1,267 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+
+# This component needs a 'ceph' user. 
+# The user should be able to run these commands with sudo without password:
+# /usr/bin/ceph-deploy
+# /usr/bin/python -c import sys;exec(eval(sys.stdin.readline()))
+# /usr/bin/python -u -c import sys;exec(eval(sys.stdin.readline()))
+# /bin/mkdir
+#
+
+package NCM::Component::Ceph::Jewel;
```

Comment:
mention in comment that this is old ceph.pm code? so i don't have to reveiw it 😄